src/js/components/atoms/label-filter/label-filter.ts

```diff
@@ -12,23 +13,22 @@ import { LabelFilterProps } from './label-filter.models';
 export class LabelFilter extends Vue implements LabelFilterProps {
     @Prop({ type: Boolean, default: false }) checked?: LabelFilterProps['checked'];
     @Prop({ type: Boolean, default: false }) disabled?: LabelFilterProps['disabled'];
+    @Prop({ type: String, required: true }) eventChange!: LabelFilterProps['eventChange'];
     @Prop({ type: String, required: true }) label!: LabelFilterProps['label'];
     @Prop({ type: String, required: true }) name!: LabelFilterProps['name'];
     @Prop({ type: String, required: true }) value!: LabelFilterProps['value'];
 
-    get payload() {
-        return {
-            checked: this.checked,
+    checkedState: boolean = this.checked || false;
+
+    emitEvent(eventName) {
+        this.$emit(eventName, {
+            checked: this.checkedState,
+            disabled: this.disabled,
             value: this.value,
-        };
+        });
     }
 
     onChange($event) {
-        const payload = {
-            ...this.payload,
-            checked: $event.target.checked,
-        };
-
-        this.$emit('label-filter-change', payload);
+        this.emitEvent(this.eventChange);
```

Comment:
Left over from when this was more complicated, `emitEvent` does not need to be its own method anymore.

Reply:
done